### DISCUSSION TITLE
Empty filter and limit

### DISCUSSION THREAD

**Discussion by echo1377** [akvorado/akvorado#297] [Q&A] [answered]
Hi

Great project.

How to set empty filter and limit = 20 in akvorado.yaml under "default-visualize-options:" ? 

It is possible to make the default sankey GraphType ?

Thank you !

**Comment by vincentbernat** [comment, marked as answer]
Default limit cannot be set yet. Same for default graph. For empty filter, use `""`.

**Reply by echo1377** [reply]
Thank you!

**Reply by vincentbernat** [reply]
It will be in the next version.

**Reply by echo1377** [reply]
Good!
